profile/README.md

```diff
@@ -1,5 +1,5 @@
-Nexus is a protocol for arbitrary length operations. These operations are modeled using a predefined set of synchronous RPCs.
+Nexus is a protocol for arbitrary duration RPC operations. These operations are modeled using a predefined set of synchronous RPCs.
```

Comment:
Reviewers: please feel free to (tell me to) revert it to "length" if you prefer that. I think "RPC operations" is helpful here to give some narrowing down of what sorts of operations the repo is concerned with.

Reply:
Agree we're already using arbitrary-duration Operations in https://docs.temporal.io/nexus